src/services/getRepeatableSetEntriesConfiguration.ts

```diff
@@ -0,0 +1,25 @@
+import { FormTypes } from '@oneblink/types'
+import { RepeatableSetElement } from '@oneblink/types/typescript/forms'
+import { FormSubmissionModel } from '../types/form'
+import { submissionService } from '@oneblink/sdk-core'
+
+export default function getRepeatableSetEntriesConfiguration(
```

Comment:
My only other suggestion here would be to just make this function a hook, and return the `useMemo` from it. Rather than having to wrap it twice inside a `useMemo` in the component.